lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
 
     # Let the XQueue know that we handled the response
     return JsonResponse({'return_code': 0})
+
+
+# pylint: disable=too-many-statements, bad-continuation
+@login_required
+def render_html_view(request):
+    """
+    This view generates an HTML representation of the specified student's certificate
+    If a certificate is not available, we display a "Sorry!" screen instead
+    """
```

Comment:
Cool, I've switched the implementation around to the data migration approach instead of JSON data -- I'd like to tackle the shift to using the configuration flag as part of the next cycle, if that's okay with you.